packages/core/core/src/AssetGraphBuilder.js

```diff
@@ -166,11 +179,27 @@ export default class AssetGraphBuilder extends EventEmitter {
       }
     }
 
+    if (this.hmrServer) {
+      this.hmrServer.updateAsset(cacheEntry);
```

Comment:
A cache entry can contain multiple assets, so probably need to split them out here?